vignettes/sim_fixed_design_custom.Rmd

```diff
@@ -0,0 +1,406 @@
+---
+title: "Custom Fixed Design Simulations: A Tutorial on Writing Code from the Ground Up"
+author: "Yujie Zhao and Keaven Anderson"
+output: rmarkdown::html_vignette
+bibliography: simtrial.bib
+vignette: >
+  %\VignetteIndexEntry{Custom Fixed Design Simulations: A Tutorial on Writing Code from the Ground Up}
+  %\VignetteEngine{knitr::rmarkdown}
+---
+
+```{r, message=FALSE, warning=FALSE}
+library(gsDesign2)
+library(simtrial)
+library(dplyr)
+library(tibble)
```

Comment:
Is this necessary to load {tibble}? I just removed it from the other vignette in https://github.com/Merck/simtrial/pull/315. It generates the NOTE:

```
* checking for unstated dependencies in vignettes ... NOTE
'library' or 'require' call not declared from: ‘tibble’
```

If you never directly call a function from {tibble}, then there is no need to load it.

Reply:
Yes, it is removable. I get it off in the new commit.